haproxy.cfg

```diff
@@ -26,38 +26,40 @@ frontend http_front
     bind *:80
     mode http
     log global
+    log-format "%ci - - [%t] \"%r\" %ST %B \"%[capture.req.hdr(1)]\" \"%[capture.req.hdr(2)]\""
 
-    capture request header host len 31
-    log-format "%ci:%cp [%t] %ft %b/%s %TR/%Tw/%Tc/%Tr/%Tt %ST %B %tsc %{+Q}r captured_host=%[capture.req.hdr(0)]"
+    redirect scheme https code 301 if !{ ssl_fc }
 
-    # Redirect HTTP to HTTPS if requesting equitytool on 80
-    acl host_equitytool req.hdr(host) -i -m beg equitytool.austinmobility.io
-    redirect scheme https code 301 if host_equitytool
 
-    # Only go to postgrest if the path is valid AND:
-    #   - either source is 10.0.0.0/8
-    #   - or source is 162.89.100.0/24 and host header is present
+frontend https_front
+    bind *:443 ssl crt /usr/local/etc/haproxy/ssl/equitytool.austinmobility.io.pem crt /usr/local/etc/haproxy/ssl/atd-postgrest.austinmobility.io.pem alpn h2,http/1.1
+    mode http
+    option httplog
+
+    capture request header host len 256
+    capture request header referer len 512
+    capture request header user-agent len 512
+
+    log-format "%ci - - [%t] \"%r\" %ST %B \"%[capture.req.hdr(1)]\" \"%[capture.req.hdr(2)]\" backend=%s/%b"
 
-    acl path_postgrest path_beg /knack-services /legacy-scripts /ctr-data-lake /parking /road-conditions /bond-reporting
+    # to use postgrest, you need to be:
+
+    # 1) using an approved path
+    # 2) using the postgrest hostname
+    # 3) be coming in from the COA network
+
+    acl path_postgrest path_beg /knack-services /legacy-scripts /parking /road-conditions /bond-reporting
     acl host_postgrest hdr(host) -i atd-postgrest.austinmobility.io
-    acl src_postgrest_internal src 10.0.0.0/8
```

Comment:
This was used to detect a request coming from the ALB when it did a health check. No need for that anymore.